src/utils/periods/relativePeriods.js

```diff
@@ -195,3 +240,118 @@ export const flattenedRelativePeriods = RELATIVE_PERIODS.reduce(
 
 export const isFixedPeriodType = periodType =>
     !!fixedPeriodTranslations[periodType]
+
+const formatYyyyMmDd = timestamp => {
+    const date = new Date(timestamp)
+    const yyyy = date.getFullYear()
+    const mm = (date.getMonth() + 1).toString().padStart(2, '0')
+    const dd = date
+        .getDate()
+        .toString()
+        .padStart(2, '0')
+
+    return `${yyyy}-${mm}-${dd}`
+}
+
+const getCurrentDate = () => new Date(Date.now())
```

Comment:
Is this not the same as just calling `new Date()`?

Reply:
This is makes it easier to mock the current date, since you then only have to mock `Date.now` and not the entire `Date` class/constructor. I was reading about this [here](https://codewithhugo.com/mocking-the-current-date-in-jest-tests/)
> Using `new Date(Date.now())` makes for code that is a lot easier to test. Mocking a function that returns a number (like `Date.now`) is a lot easier than mocking a constructor.

Reply:
Ah I see. Maybe a comment explaining that would be helpful. Otherwise it's easy to assume for someone not familiar with the app that it can be factored out (if you don't look at the tests).

Reply:
Yeah I agree a comment would be useful, added in d97053b0ad8a0916c627156a7591d4b682c15d7e. Having said that, probably it would become evident that this could not be factored out pretty quickly, since the tests would start failing.